crates/sui-indexer-alt-graphql/src/lib.rs

```diff
@@ -278,6 +282,17 @@ pub async fn start_rpc(
     let rpc = RpcService::new(args, version, schema(), registry, cancel.child_token());
     let metrics = rpc.metrics();
 
+    // Create gRPC client for transaction execution/simulation
+    let grpc_client = if let Some(url) = &tx_exec_full_node.full_node_rpc_url {
```

Comment:
Consider creating a wrapper around this client in the `sui-indexer-alt-reader` crate, similar to what I did for the consistent store, to take advantage of the null object pattern (and also to make it easier to potentially re-use this work in jsonrpc-alt).